.github/workflows/release.yml

```
@@ -90,14 +90,14 @@ jobs:
     needs: [publish_celery_crate]
     if: startsWith(github.ref, 'refs/tags/')
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
```

The "Generate release notes" step below requires history for tags and commits. So I think we need:
```suggestion
      - uses: actions/checkout@v3
        with:
          fetch-depth: 0
```

https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches